ml/backend/ggml/ggml.go

```diff
@@ -39,7 +40,7 @@ func devices() []*C.struct_ggml_backend_device {
 }
 
 type Backend struct {
-	meta    *fs.GGML
+	meta    *fsggml.GGML
```

Comment:
Do we want to tie the model format to the backend like this? Ideally backends can transparently load from `ggml` or other formats later

Reply:
Ideally yes but that can be an improvement later when there are more file formats and backends